pimd/subdir.am

```diff
@@ -171,17 +171,15 @@ clippy_scan += \
 pimd_pimd_CFLAGS = $(AM_CFLAGS) -DPIM_IPV=4
 pimd_pimd_LDADD = lib/libfrr.la $(LIBCAP)
 
-if PIMD
-if DEV_BUILD
+if PIM6D
 #
 # pim6d is only enabled for --enable-dev-build, and NOT installed currently
```

Comment:
Do we need to change this comment too?

Reply:
Thats a good point :) Thanks @ton31337 . I have removed the comment.